pom.xml

```diff
@@ -167,7 +167,7 @@
                     <plugin>
                         <groupId>org.jvnet.jaxb2.maven2</groupId>
                         <artifactId>maven-jaxb2-plugin</artifactId>
-                        <version>0.12.3</version>
+                        <version>0.14.0</version>
```

Comment:
Maybe set forceRegenerate to false below? https://github.com/highsource/maven-jaxb2-plugin/wiki/Do-Not-Use-forceRegenerate

Reply:
Nice catch! Build time down from 1m to 9s when I run "mvn install -Pgenerate-from-wsdl" two times in a row!